### ISSUE TITLE
[DOCUMENT]: GIBCT Section 116 - Excel document has merged cells that could cause accessibility issues

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#3835]
## Description

VA 508 office has identified the Excel download has some merged cells that are not allowing proper structure and relationships to be determined with screen readers. This is a violation of WCAG 1.3.1. Screenshot attached below.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## Remediation Resources
* https://accessibility.psu.edu/microsoftoffice/excel/#region
* Please follow up with Trevor next week if you have questions or issues remediating. The VA 508 office has offered to help us understand the nature of these errors and I can arrange a meeting if you'd like.

## Acceptance Criteria

* Screen readers should announce the cells as table cells, with proper column headings announced
* "Merged" cells should announce as a table row

## WCAG or Vendor Guidance (optional)

* [Info and Relationships: Understanding SC 1.3.1](https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-programmatic.html)

## Screenshots or Trace Logs

![merged-cells](https://user-images.githubusercontent.com/934879/69807290-8ca5a580-11aa-11ea-9fa4-68d2bba76507.jpg)

**Comment by desireeturner** [comment]
@TamGIBill here's another one that I believe falls into your wheelhouse. Let me know if we can assist!

**Comment by desireeturner** [comment]
@TamGIBill Is this something that has been resolved? If so, I can assign it to Trevor to re-test. Thanks!

**Comment by desireeturner** [comment]
Reached out to @TamGIBill about this via email today.

**Comment by desireeturner** [comment]
@1Copenut This has been updated and is ready for your review. Thank you!

**Comment by 1Copenut** [comment]
@desireeturner I took a look at this Excel document with the Windows Excel accessibility checker and did a manual spot check. The cells are no longer merged and not showing any warnings. I think we can move this one to close. Thank you!
